### DISCUSSION TITLE
Using freud's SolidLiquid analysis method

### DISCUSSION THREAD

**Discussion by Harsh-hkr** [glotzerlab/hoomd-blue#1892] [Support]
Hi, I am trying to replicate the results from Hoomd tuturial - https://github.com/glotzerlab/hoomd-examples/blob/trunk-minor/00-Introducing-HOOMD-blue/07-Analyzing-Trajectories.ipynb

This is a copy of the file which I worked on:
https://github.com/Harsh-hkr/hoomd/blob/main/07-Analyzing-Trajectories%20(1).ipynb 

I am having trouble understanding why the SolidLiquid analysis yields a completely different result.

**Comment by joaander** [comment]
Thanks for sharing. Did you have a specific question you would like answered? If yes, please let us know what the question is.

**Reply by Harsh-hkr** [reply]
Hi, I apologize if my previous question was unclear. My question is why do I get completely different results for the Solid Liquid analysis when I am using the same tutorial code ie. my graphs and rendered simulations vary drastically from the tutorial 07. Is there something I am missing? Or is the system not at equilibrium and needs to be run for more timesteps?

**Reply by joaander** [reply]
Yes, you may need to run longer and/or try again with a different random number seed. As the tutorial says:
> Nucleation is a rare event, so you need to keep running the simulation until it occurs. If you ran this simulation many times with different random seeds, each would take a different number of steps to nucleate. You need to examine the simulation trajectory in detail to determine if you have run it long enough.

Each time I rerun that tutorial with a new version of HOOMD-blue, I typically need to try several different seeds before I find one that nucleates quickly.

**Reply by Harsh-hkr** [reply]
Thanks a lot!
